geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -467,10 +582,14 @@ app.DrawController.createStyleFunction_ = function() {
     }
 
     // goog.asserts.assert(goog.isDef(this.get('__style__'));
-    var color = this.get('color');
+    var color = this.get('color') ? this.get('color') : '#FF0000';
     var rgb = goog.color.hexToRgb(color);
+    var opacity = this.get('opacity');
+    if (!goog.isDefAndNotNull(opacity)) {
```

Comment:
May `opacity` be `null`. I'd recommend being as strict as possible with types. If you know that `opacity` is either a `number` or `undefined` you should use:

```js
if (!goog.isDef(opacity)) {
  opacity = 1;
}
```